src/kernels/common/baseJupyterSession.ts

```diff
@@ -243,85 +208,6 @@ export abstract class BaseJupyterSession<T extends 'remoteJupyter' | 'localJupyt
         this.shutdownSession(oldSession, undefined, false).catch(noop);
     }
 
-    public requestExecute(
-        content: KernelMessage.IExecuteRequestMsg['content'],
-        disposeOnDone?: boolean,
-        metadata?: JSONObject
-    ): Kernel.IShellFuture<KernelMessage.IExecuteRequestMsg, KernelMessage.IExecuteReplyMsg> {
-        if (!this.session?.kernel) {
-            throw new SessionDisposedError();
-        }
-        // Make sure to chain the executes
-        return this.chainingExecute.requestExecute(this.session.kernel, content, disposeOnDone, metadata);
-    }
-
-    public requestDebug(
-        content: KernelMessage.IDebugRequestMsg['content'],
-        disposeOnDone?: boolean
-    ): Kernel.IControlFuture<KernelMessage.IDebugRequestMsg, KernelMessage.IDebugReplyMsg> {
-        if (!this.session?.kernel) {
-            throw new SessionDisposedError();
-        }
-        return this.session.kernel.requestDebug(content, disposeOnDone);
-    }
-
-    public requestInspect(
-        content: KernelMessage.IInspectRequestMsg['content']
-    ): Promise<KernelMessage.IInspectReplyMsg> {
-        if (!this.session?.kernel) {
-            throw new SessionDisposedError();
-        }
-        return this.session.kernel.requestInspect(content);
-    }
-
-    public requestComplete(
-        content: KernelMessage.ICompleteRequestMsg['content']
-    ): Promise<KernelMessage.ICompleteReplyMsg> {
-        if (!this.session?.kernel) {
-            throw new SessionDisposedError();
-        }
-        return this.session.kernel.requestComplete(content);
-    }
-
-    public sendInputReply(content: KernelMessage.IInputReply) {
-        if (this.session && this.session.kernel) {
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            this.session.kernel.sendInputReply(content);
-        }
-    }
-
-    public registerCommTarget(
-        targetName: string,
-        callback: (comm: Kernel.IComm, msg: KernelMessage.ICommOpenMsg) => void | PromiseLike<void>
-    ) {
-        if (this.session && this.session.kernel) {
```

Comment:
These are just wrappers, the property `session.kernel` is actually a public property
Hence remove these wrappers in favour of directly using the `session.kernel`, which is a JupyterLab API.
I.e. fewer wrappers.